.gitlab-ci.yml

```diff
@@ -54,6 +54,36 @@ tox:
     - !reference [.load_github_key, script]
     - tox -e py
 
+# used by release
+.python_versions:
+  parallel:
+    matrix:
+      - BUILD_PY_VERSION: [python3.6]
+      - RELEASE_PY_VERSION: [python3.6]
+
+release:
+  before_script:
+    - git fetch --unshallow
+
+get_version:
+  stage: test
+  tags:
+    - dind
+  image: ${BASE_CONTAINER_REGISTRY}/python3.8-builder:${BASE_CONTAINER_VERSION}
+  before_script:
+    - git fetch --unshallow
+  script:
+    - pip install importlib-metadata
+    - pip install --no-deps .
+    - >
+      python -c "from importlib_metadata import version; print(version('indexd'))"
+      > VERSION.txt
+    - cat VERSION.txt
```

Comment:
Not familiar with this ... will this VERSION.txt file be used for?

Reply:
to provide version for label in docker image.